pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -134,7 +135,9 @@ type ACMEIssuer struct {
 	DNS01 *ACMEIssuerDNS01Config `json:"dns01,omitempty"`
 }
 
+// ACMEIssuerHTTP01Config is a structure containing the ACME HTTP configuration options
 type ACMEIssuerHTTP01Config struct {
+	SolverServiceType corev1.ServiceType `json:"solverServiceType,omitempty"`
```

Comment:
Can we call this `serviceType` instead? I think the fact this is for the solver is inferred by the fact that we're configuring the `issuer.spec.acme.http01` stanza.